### ISSUE TITLE
Geocaching.com Messagecenter Support (Feature Request)

### ISSUE THREAD

**Issue by basti2k** [cgeo/cgeo#4917]
I would be glad if CGEO support the Message Center on geocaching.com.
The app could check for new messages when you get anything downloaded from geocaching.com.
Since the number of new messages is a part of every page.
![gc-mess](https://cloud.githubusercontent.com/assets/1093104/7367671/7044c60a-eda1-11e4-8dcc-04c9f296ece4.png)

**Comment by Natureshadow** [comment]
I do not think that c:geo is a chat app (ok, Geocaching.com is not a chat platform, in the first place).

The feature would make interfacing with Geocaching.com even more complicated, draw resources and the like.

If you want all features of the website, please use the website. c:geo is a Geocaching app.

JM2C,
Nik

**Comment by rsudev** [comment]
I think full chat integration was not suggested here (and that clearly would leave the scope of what I would like to see in c:geo as well), but to just take advantage of the already present information in the pages we read and signal that to the user (possible offering also a link to jump there directly).
Something like that seems at least feasible from a feature perspective (even though I currently have no real idea where to present that information).
This does not imply that I consider working on such a feature, as I see more pressing concerns elsewhere.

**Comment by basti2k** [comment]
It would only be useful if you do not query the status of the messagecenter permanently. Only if I'm doing anything in the app (load a cache, etc.).
It would be enough that one gets displayed a link to the messagecenter.

**Comment by Lineflyer** [comment]
I think the balance of work to effort is too high to implement something like this as long as we do not have an efficient way to communicate with the server (e.g. API).
Furthermore c:geo is not a messaging app but a geocaching app. Maybe there might be another developer creating a dedicated app for that or Groundspeak will have such an app...

**Comment by Bananeweizen** [comment]
I actually explicitly want c:geo to NOT support the message center. Just read about what most cache _owners_ think of it. Every earth cache owner now needs to log into the website to read the answers that a user sent, because you can't read the message center messages via mail. And saying that c:geo would reduce this burden is totally not true, as there are many people outside who don't use it. Head over to the groundspeak forum, if you need some more details about this...

**Comment by mucek4** [comment]
It's not really our problem if GS can't make notifications right. Or customizable.

**Comment by SammysHP** [comment]
I'm also for not implementing this in c:geo. If someone needs it and can develop a small app it should be easy to create one that polls every few minutes and shows a notification in the status bar.

**Comment by indy-bikes** [comment]
After struggling with sending messages with my phone through the website, I searched out this forum to see if this was being looked at. I think it might make it easier to communicate with other players of the game while in the field.(read 'help')
I don't know if this is the best place to get help in this way, but I am looking for a better way to use the message feature. Just wanted to let you know that others may also be looking for such a thing. I wish that I was able to help create such an app, but alas, I am no good with programing.
Thanks,
